x-pack/test/api_integration/apis/osquery/packs.ts

```diff
@@ -101,7 +99,15 @@ export default function ({ getService }: FtrProviderContext) {
           description: '123',
           id: '123',
         });
-      packagePolicyId = packagePolicy.id;
+
+      if (!packagePolicyResponse.body.item) {
+        // eslint-disable-next-line no-console
+        console.error({ MISSING: packagePolicyResponse });
```

Comment:
@tomsonpl In the future, please use `getService('log')` to get the logger so that logs are properly associated with tests and failures.